integrations/server/test_covidcast.py

```diff
@@ -307,28 +317,30 @@ def fetch(geo_value):
 
       return response
 
+    self.maxDiff = None
     # test fetch a specific region
-    r = fetch('11111')
+    r = fetch(msa[0])
     self.assertEqual(r['message'], 'success')
     self.assertEqual(r['epidata'], expected[0:1])
     # test fetch a specific yet not existing region
-    r = fetch('55555')
-    self.assertEqual(r['message'], 'no results')
+    r = fetch('11111')
+    self.assertEqual(r['message'], 'invalid geo_value for the requested geo_type')
+
     # test fetch multiple regions
-    r = fetch('11111,22222')
+    r = fetch('{},{}'.format(msa[0], msa[1]))
     self.assertEqual(r['message'], 'success')
     self.assertEqual(r['epidata'], expected[0:2])
     # test fetch multiple noncontiguous regions
-    r = fetch('11111,33333')
+    r = fetch('{},{}'.format(msa[0], msa[2]))
     self.assertEqual(r['message'], 'success')
     self.assertEqual(r['epidata'], [expected[0], expected[2]])
     # test fetch multiple regions but one is not existing
-    r = fetch('11111,55555')
-    self.assertEqual(r['message'], 'success')
-    self.assertEqual(r['epidata'], expected[0:1])
+    r = fetch('{},11111'.format(msa[0]))
+    self.assertEqual(r['message'], 'invalid geo_value for the requested geo_type')
```

Comment:
great usage!!

Reply:
thanks for the comments! will do some revisions on these soon. :)